src/components/Auth/AuthForm.tsx

```diff
@@ -0,0 +1,159 @@
+import classNames from 'classnames';
+import React, { useEffect, useState } from 'react';
+import { createUser, getUserByEmail } from '../../api/users';
+import { User } from '../../types/User';
+
+export type Props = {
+  onLogin: (user: User) => void,
+};
+
+export const AuthForm: React.FC<Props> = ({ onLogin }) => {
+  const [email, setEmail] = useState('');
+  const [name, setName] = useState('');
+  const [needToRegister, setNeedToRegister] = useState(false);
+  const [loading, setLoading] = useState(false);
```

Comment:
```suggestion
  const [isLoading, setIsLoading] = useState(false);
```
![image](https://user-images.githubusercontent.com/96409563/218992825-78054016-11ae-4cf0-97cc-886b81507df3.png)